docs/AccelNNPAHowToUseAndTest.md

```diff
@@ -0,0 +1,29 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
```

Comment:
Big big thanks for adding doc on how to use NNPA. 

I would suggest that you add links to that page from the CONTRIBUTING (just add "handling and testing for specialized accelerator, for example) as well as near the top of /docs/Testing.md, for example "Testing of specific accelerators are described [here](AccelNNPAHowToUseAndTest.md)"

Reply:
Done. Added some sentences in `CONTRIBUTING.md` and `docs/Testing.md`